File: app/src/main/java/com/example/imagerecognitionsystem/MainActivity.java
package com.example.imagerecognitionsystem;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {
    private Button getStarted;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        getStarted = (Button) findViewById(R.id.getStarted);
        getStarted.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View v) {
                openMainApp();
            }

        });
    }

    public void  openMainApp() {
        Intent intent = new Intent(this, mainAppPage.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/example/imagerecognitionsystem/MainActivity.kt
package com.example.imagerecognitionsystem

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private var getStarted: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        getStarted = findViewById<View?>(R.id.getStarted) as Button?
        getStarted?.setOnClickListener(View.OnClickListener { openMainApp() })
    }

    fun openMainApp() {
        val intent = Intent(this, mainAppPage::class.java)
        startActivity(intent)
    }
}